client/src/components/Item/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+
+class Item extends Component {
+  constructor(props) {
+    super(props);
+    this.clickHandler = this.clickHandler.bind(this);
+  }
+
+  //invoke
+  clickHandler() {
+    console.log('this.props.id is :', this.props.id)
+    this.props.cancelItem(this.props.id)
+  }
+
+  render() { 
+    const name = this.props.name;
+    const category = this.props.category;
+    const id = this.props.id;
```

Comment:
I would try to use deconstruction here to declare all these variables.
Since they are all coming from this.props, see if this would work:
const { name, category, id } = this.props